dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/ContainerClassLoader.java

```diff
@@ -1628,7 +1831,10 @@ protected String findLibrary(String libName) {
         return null;
     }
 
-    protected ByteResourceInformation findClassBytes(String className, String resourceName, ClassLoaderHook hook) throws IOException {
+    // TODO renamed this method with Impl so we can continue to test what happens when this method throws
+    // an IOException in the unit tests with a method override. If it wasn't for the unit test I would
+    // have put the IOException handling directly into this method from AppClassLoader
```

Comment:
```suggestion
    // Renamed this method with an Impl name so we can continue to test what happens when this method throws
    // an IOException in the unit tests with a method override. If it wasn't for the unit test it would be nice to
    // put the IOException handling directly into this method from AppClassLoader.
    // TODO consider mocking smartClassPath.getByteResourceInformation to throw an IOException for the test instead
```